Source/Core/Core/Brawlback/Savestate.cpp

```diff
@@ -222,7 +222,10 @@ void BrawlbackSavestate::initBackupLocs()
         for (auto& loc : this->backupLocs) {
             u32 size = loc.endAddress-loc.startAddress;
             double newsize = ((double)size / 1000.0) / 1000.0;
+#ifdef _WIN32
+            // TODO: fix format since This keeps crashing on compilation for macosx because of %p
```

Comment:
Oops! That's my bad, I overwrote your change from before, didn't realize this caused crashes on mac. Would you mind putting the fix you had earlier back in?

Reply:
other than that, lgtm

Reply:
Done!